File: lib/py/uapi/Message.py
from typing import cast


class Message:
    def __init__(self, header: dict[str, object], body: dict[str, object]):
        self.header = header.copy()
        self.body = body

    def get_body_target(self) -> str:
        entry = next(iter(self.body.items()))
        return entry[0]

    def get_body_payload(self) -> dict[str, object]:
        entry = next(iter(self.body.items()))
        return cast(dict[str, object], entry[1])

File: lib/ts/uapi/Message.ts
export class Message {
    constructor(
        public header: Record<string, any>,
        public body: Record<string, any>,
    ) {}

    getBodyTarget(): string {
        const entry = Object.entries(this.body)[0];
        return entry[0];
    }

    getBodyPayload(): Record<string, any> {
        const entry = Object.entries(this.body)[0];
        return entry[1] as Record<string, any>;
    }
}
